node/src/reducer.rs

```diff
@@ -17,7 +17,18 @@ pub fn reducer(
     let meta = action.meta().clone();
     match action.action() {
         Action::CheckTimeouts(_) => {
-            if state.p2p.ready().is_some() {
+            if let Some(p2p) = state.p2p.ready() {
+                if let Some(kad_state) = &p2p.network.scheduler.discovery_state {
+                    if p2p.ready_peers().is_empty() && kad_state.has_bootstraped {
+                        if let Some(tip) = &state.transition_frontier.best_tip_breadcrumb() {
+                            // TODO: this might need to change in future
+                            if tip.height() == 296372 {
```

Comment:
Where does this value come from?

Reply:
To anyone reading this code, this is effectively a random number as there is no explanation or description. In general, one should not have any numbers like this in the middle of a file but use a named constant instead. I don't know the best way to name, this, but anything is better than no name. 

```rust
const KNOWN_PROBLEMATIC_BLOCK_HEIGHT: usize = 296_372;
// ...
if tip_height == KNOWN_PROBLEMATIC_BLOCK_HEIGHT {
// ...
```